modern/src/index.js

```diff
@@ -1,4 +1,5 @@
-import 'typeface-roboto';
+import '@fontsource/roboto/latin-400.css';
+import '@fontsource/roboto/latin-500.css';
```

Comment:
What about 300 and 700?

Reply:
It seemed that only 400 and 500 were used in traccar, but it turns out that 400, 500, and 700 are used. However 300 doesn't seem to be used, is it?